Scripts/Restriction/ranacompiler.py

```diff
@@ -102,20 +104,12 @@ def regex(site):
 
 
 def is_palindrom(sequence):
-    """is_palindrom(sequence) -> bool.
-
-    True is the sequence is a palindrom.
-    sequence is a Seq object.
-    """
+    """Is the sequence (Seq objecT) as a palindrom."""
```

Comment:
That would be good. Something like:

```
def is_palindrom(...):
    # ... deprecation notice here? I'm not sure if this is overkill since this is a small function ... 
    return is_palindrome(...)

def is_palindrom(...):
    # actual function
```